MobileCenter/MobileCenter/MSMobileCenter.m

```diff
@@ -11,9 +12,7 @@
 static MSMobileCenter *sharedInstance = nil;
 static dispatch_once_t onceToken;
 
-/**
- * Base URL for HTTP Ingestion backend API calls.
- */
+// Base URL for HTTP Ingestion backend API calls.
```

Comment:
This is a doc. Please revert. We use // for single line and /* */ for multiple lines but /** */ is for docs.